packages/storybook/src/spright/chat/conversation/chat-conversation-matrix.stories.ts

```diff
@@ -218,6 +218,31 @@ export const conversationWithInputSizing: StoryFn = createMatrixThemeStory(html`
     ${createMatrix(conversationWithInput, [heightStates])}
 `);
 
+export const conversationBackground: StoryFn = createStory(
```

Comment:
Could you make this a matrix test that includes the theme and the hide-background state?

It'd also be nice to copy the `<p>` styling from above.

```
    <p 
        style="
        font: var(${bodyFont.cssCustomProperty});
        color: var(${bodyFontColor.cssCustomProperty});
        margin-bottom: 0px;
        "
    >
```

Reply:
I think I have it the way you expect?